roles/optional-operators-subscribe/templates/optional-operators-subscribe-commands.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
```

Comment:
I assume this file is almost the same as in project.
Can we add a comment with reference to the source?
Just in case sb would like to keep them in sync.